test/cli/package-test-simple/package-test-simple.out

```diff
@@ -0,0 +1,11 @@
+main_lib/lib.rb:8: Unable to resolve constant `TestOnly` https://srb.help/5002
+     8 |  Project::TestOnly::SomeHelper.new
+          ^^^^^^^^^^^^^^^^^
+  Autocorrect: Use `-a` to autocorrect
+    main_lib/lib.rb:8: Replace with `Project::TestOnly`
```

Comment:
😬 this looks like a bug?

Reply:
The autocorrects need a lot more work (they should be suggesting imports/exports), but that seems like too much for this PR. I'd prefer to tackle in a follow-up.